modules/spotxBidAdapter.js

```diff
@@ -149,7 +149,7 @@ export const spec = {
         }
       }
 
-      const mimes = getBidIdParameter('mimes', bid.params) || ['application/javascript', 'video/mp4', 'video/webm'];
+      const mimes = deepAccess(bid, 'mediaTypes.video.mimes') || getBidIdParameter('mimes', bid.params) || ['application/javascript', 'video/mp4', 'video/webm'];
```

Comment:
How are you choosing which adapters to update - and do we need to run this through them? Here for example I think the changes might be breaking; I can imagine a publisher that is using spotx but not this new video module, with video adUnits that would work before, but not after, this change.

Reply:
This change is not specific to the video module. We have been pushing SSPs to read standard oRTB video params from the `mediaTypes.video` instead of from `bid.params`. This way the publisher can define the params in one place, instead of copying them for every SSP. 
https://docs.prebid.org/dev-docs/adunit-reference.html#adunitmediatypesvideo

Reply:
Agreed, `mediaTypes.video` is the "right" way, but what I'm not sure about is the precedence; I think if this was picking from  `bid.params` falling back to `mediaTypes.video` it would be OK; but the other way around can potentially break some publishers. 

The former can only add to the video request, and would keep all existing fields unchanged. The latter can potentially change them if you set up the adUnit in some (admittedly weird) ways.

Reply:
I see your point; I agree and will address

Reply:
@dgirardi this has been addressed